src/OpenTelemetry/Metrics/ExplicitBucketHistogramConfiguration.cs

```diff
@@ -24,6 +24,8 @@ public class ExplicitBucketHistogramConfiguration : MetricStreamConfiguration
         /// </summary>
         /// <remarks>
         /// The array must be in ascending order with distinct values.
+        /// Empty array means no histogram buckets will be generated.
```

Comment:
Rephrase "means" -> "would result in"
Something like:

- An empty array would result in no histogram buckets being used. 
- Null value would result in default histogram buckets being used.